config.py

```diff
@@ -57,7 +57,8 @@ def network_configuration():
         "real_machine_identifier_name": "stockholm_server_1",  # can be anything e.g. real_machine_ip_address, name, etc
         "ignore_real_machine_ip_addresses": list(set([real_machine_ip_address, "127.0.0.1"])),
         # e.g. ["10.0.0.1", "192.168.1.1"]
-        "ignore_real_machine_ports": []  # e.g. [22, 80, 5000]
+        "ignore_real_machine_ports": [],  # e.g. [22, 80, 5000]
+        "timeout": 3600 # Default value
```

Comment:
be more clear than `timeout`, maybe `split_pcap_file_timeout` or something similar.

Reply:
Okay